acstools/acszpt.py

```diff
@@ -153,6 +153,7 @@ def __init__(self, date, detector, filt=None):
                     'F140LP', 'F150LP', 'F165LP']
         }
         self._zpt_table = None
+        self._warnings = None
```

Comment:
Probably easier if you keep it as a list throughout.
```suggestion
        self._warnings = []
```